package.json

```diff
@@ -27,11 +27,11 @@
   "dependencies": {
     "@expo/vector-icons": "^6.3.1",
     "axios": "^0.18.0",
-    "expo": "^26.0.0",
+    "expo": "^26.1.1",
     "firebase": "^4.13.1",
     "prop-types": "^15.6.1",
     "react": "16.3.0",
-    "react-native": "~0.54.1",
+    "react-native": "https://github.com/expo/react-native/archive/sdk-26.0.0.tar.gz",
```

Comment:
Can you make a note in the trello to check for any dependency changes or breaking changes in the update from 0.54 to sdk 26?